pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -151,15 +173,18 @@ var FileAutoComplete = React.createClass({
         return changed;
     },
 
-    finishUpdate: function(result, error) {
-        result = result.sort(function(a, b) {
-            return a.path.localeCompare(b.path,
-                                        { sensitivity: 'base'});
+    finishUpdate: function(directory, results, error) {
```

Comment:
Why aren't we getting directory from state?

Reply:
I left it there because of my earlier buggy version - removed